smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy

```diff
@@ -309,3 +329,94 @@ structure StructureListMember {
+    {
+        id: "RpcV2CborSparseMapsSerializeNullValues"
+        documentation: "Serializes null values in maps"
+        protocol: rpcv2Cbor
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        },
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=v29zcGFyc2VTdHJpbmdNYXC%2FY2Zvb%2Fb%2F%2Fw%3D%3D
+        body: "v29zcGFyc2VTdHJpbmdNYXC/Y2Zvb/b//w=="
+        params: {
+            "sparseStringMap": {
+                "foo": null
+            }
+        }
+        method: "POST"
+        uri: "/service/RpcV2Protocol/operation/SparseNullsOperation",
+    },
+    {
+        id: "RpcV2CborSparseListsSerializeNull"
+        documentation: "Serializes null values in lists"
+        protocol: rpcv2Cbor
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        },
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=v3BzcGFyc2VTdHJpbmdMaXN0n%2Fb%2F%2Fw%3D%3D
+        body: "v3BzcGFyc2VTdHJpbmdMaXN0n/b//w=="
+        params: {
+            "sparseStringList": [
+                null
+            ]
+        }
+        method: "POST"
+        uri: "/service/RpcV2Protocol/operation/SparseNullsOperation",
+    }
+])
+@httpResponseTests([
+    {
+        id: "RpcV2CborSparseMapsDeserializeNullValues"
+        documentation: "Deserializes null values in maps"
+        protocol: rpcv2Cbor,
+        code: 200,
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=v29zcGFyc2VTdHJpbmdNYXC%2FY2Zvb%2Fb%2F%2Fw%3D%3D
+        body: "v29zcGFyc2VTdHJpbmdNYXC/Y2Zvb/b//w=="
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Content-Type": "application/cbor"
+        },
+        params: {
+            "sparseStringMap": {
+                "foo": null
+            }
+        }
+    }
+    {
+        id: "RpcV2CborSparseListsDeserializeNull"
+        documentation: "Deserializes null values in lists"
+        protocol: rpcv2Cbor,
+        code: 200,
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=v3BzcGFyc2VTdHJpbmdMaXN0n%2Fb%2F%2Fw%3D%3D
+        body: "v3BzcGFyc2VTdHJpbmdMaXN0n/b//w=="
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Content-Type": "application/cbor"
+        },
+        params: {
+            "sparseStringList": [
+                null
+            ]
+        }
+    }
+])
+operation SparseNullsOperation {
+    input: SparseNullsOperationInputOutput
+    output: SparseNullsOperationInputOutput
+}
+
+structure SparseNullsOperationInputOutput {
+    sparseStringList: SparseStringList
+    sparseStringMap: SparseStringMap
```

Comment:
The tests for sparse maps that are currently in the `cbor-maps.smithy` file should be pulled out to this operation as well.

Reply:
Have moved them into a separate operation